apps/admin/frontend/src/screens/election_screen.tsx

```diff
@@ -61,23 +61,17 @@ export function ElectionScreen(): JSX.Element {
           </P>
         </div>
       </ElectionCard>
-      {isSystemAdministratorAuth(auth) && (
-        <UnconfigureMachineButton
-          isMachineConfigured
-          unconfigureMachine={unconfigureMachine}
-        />
-      )}
-      {isElectionManagerAuth(auth) && (
-        <React.Fragment>
-          <P>
-            Save the election package to the USB drive to configure VxSuite
-            components.
-          </P>
-          <P>
-            <ExportElectionPackageModalButton />
-          </P>
-        </React.Fragment>
-      )}
+      <div style={{ display: 'flex', gap: '1rem' }}>
+        {(isSystemAdministratorAuth(auth) || isElectionManagerAuth(auth)) && (
+          <ExportElectionPackageModalButton />
+        )}
+        {isSystemAdministratorAuth(auth) && (
+          <UnconfigureMachineButton
+            isMachineConfigured
+            unconfigureMachine={unconfigureMachine}
+          />
+        )}
+      </div>
```

Comment:
This implementation is for Option 2. Review commit by commit to compare Option 1 and Option 2.

Reply:
I think it's clearer without the text. I'd be curious to see what it's like with the "unconfigure machine" button aligned to the right edge of the election card

Reply:
<img width="1011" alt="Screenshot 2024-09-11 at 2 58 17 PM" src="https://github.com/user-attachments/assets/f51701de-f752-4a23-87aa-b673e2afe1e3">

I think it's a bit less common of an alignment for buttons, but interested in what you think

Reply:
And would be fine with removing the text for Election Managers as well, or is it more helpful for them?

Reply:
Ok yeah, let's stick with them both on the left. Sometimes I like to put destructive actions a bit out of the main flow, but it doesn't seem necessary here.

Reply:
> And would be fine with removing the text for Election Managers as well, or is it more helpful for them?

I think it's best to remove the text in both and have them consistent

Reply:
Oh right, that's a helpful thing to keep in mind reg. destructive actions. Will revert to left-aligned buttons for this.